lib/galaxy/datatypes/sequence.py

```diff
@@ -20,7 +20,7 @@
 from galaxy.datatypes.metadata import MetadataElement
 from galaxy.datatypes.sniff import get_headers
 from galaxy.util import nice_size
-from galaxy.util.checkers import is_gzip
+from galaxy.util.checkers import is_gzip, is_bz2
```

Comment:
The important bit is that they should be alphabetically ordered (`is_bz2` should be before `is_gzip`), so you can also use:
```
from galaxy.util.checkers import is_bz2, is_gzip
```
The one I prefer is the second option proposed by @mvdbeek because it avoids repetions and is easier to deal with in case of merge conflicts.